CHANGELOG

```diff
@@ -5,6 +5,28 @@ The following overview lists user facing changes as well as newly added
 features in inverse chronological order.
 
 
+REMOVED: nutils configuration file
```

Comment:
Capitalize Nutils? (There are three occurrences in the CHANGLOG patch.)

Reply:
fixed